cumulus/parachains/runtimes/coretime/coretime-westend/src/coretime.rs

```diff
@@ -216,6 +225,12 @@ impl CoretimeInterface for CoretimeAllocator {
 	) {
 		use crate::coretime::CoretimeProviderCalls::AssignCore;
 
+		// Weight for `assign_core` from westend benchmarks:
+		// `ref_time` = 10177115 + (1 * 25000000) + (2 * 100000000) + (57600 * 13932) = 937660315
+		// `proof_size` = 3612
+		// Add 5% to each component and round to 2 significant figures.
+		let call_weight = Weight::from_parts(980_000_000, 3800);
```

Comment:
Where are those numbers coming from?
Can we replace this with formula withreal weights? I mean something like: 
```
let call_weight = Weight::from_parts(T::WeightInfo::what_ever_benchmarks2().ref_time + T::WeightInfo::what_ever_benchmarks2().ref_time...., T::WeightInfo::what_ever_benchmarks1().proof_size + T::WeightInfo::what_ever_benchmarks2().proof_size...)
```
Or can we just use `Weight::from_parts(1_000_000_000, 200_000)` as const everywhere?

Reply:
These numbers were there before, I got them from reverting https://github.com/paritytech/polkadot-sdk/pull/6228